src/uncrustify.cpp

```diff
@@ -2099,17 +2101,18 @@ struct lang_name_t
 
 static lang_name_t language_names[] =
 {
-   { "C",    LANG_C             },
-   { "CPP",  LANG_CPP           },
-   { "D",    LANG_D             },
-   { "CS",   LANG_CS            },
-   { "VALA", LANG_VALA          },
-   { "JAVA", LANG_JAVA          },
-   { "PAWN", LANG_PAWN          },
-   { "OC",   LANG_OC            },
-   { "OC+",  LANG_OC | LANG_CPP },
-   { "CS+",  LANG_CS | LANG_CPP },
-   { "ECMA", LANG_ECMA          },
+   { "C",        LANG_C                        },
+   { "CPP",      LANG_CPP                      },
+   { "D",        LANG_D                        },
+   { "CS",       LANG_CS                       },
+   { "VALA",     LANG_VALA                     },
+   { "JAVA",     LANG_JAVA                     },
+   { "PAWN",     LANG_PAWN                     },
+   { "OC",       LANG_OC                       },
+   { "OC+",      LANG_OC | LANG_CPP            },
+   { "CS+",      LANG_CS | LANG_CPP            },
+   { "ECMA",     LANG_ECMA                     },
+   { "C-Header", LANG_OC | LANG_CPP | FLAG_HDR },
```

Comment:
Awesome 💯. We actually had a hack in our code to treat all header files as `OC+` as there was no way to tell them apart. I think this change needs to be documented in the PR description. cc @Rassi